plugins/chat/assets/javascripts/discourse/models/chat-thread.js

```diff
@@ -0,0 +1,78 @@
+import RestModel from "discourse/models/rest";
+import I18n from "I18n";
+import User from "discourse/models/user";
+import { escapeExpression } from "discourse/lib/utilities";
+import { tracked } from "@glimmer/tracking";
+
+export const THREAD_STATUSES = {
+  open: "open",
+  readOnly: "read_only",
+  closed: "closed",
+  archived: "archived",
+};
+
+export function threadStatusName(status) {
+  switch (status) {
+    case THREAD_STATUSES.open:
+      return I18n.t("chat.thread_status.open");
+    case THREAD_STATUSES.readOnly:
+      return I18n.t("chat.thread_status.read_only");
+    case THREAD_STATUSES.closed:
+      return I18n.t("chat.thread_status.closed");
+    case THREAD_STATUSES.archived:
+      return I18n.t("chat.thread_status.archived");
+  }
+}
+
+const READONLY_STATUSES = [
+  THREAD_STATUSES.closed,
+  THREAD_STATUSES.readOnly,
+  THREAD_STATUSES.archived,
+];
+
+const STAFF_READONLY_STATUSES = [
+  THREAD_STATUSES.readOnly,
+  THREAD_STATUSES.archived,
+];
+
+export default class ChatThread extends RestModel {
+  @tracked title;
+  @tracked status;
+
+  get escapedTitle() {
+    return escapeExpression(this.title);
+  }
+
+  get isOpen() {
+    return this.status === THREAD_STATUSES.open;
+  }
```

Comment:
Is it expected in english to have "open" and then "closed"? Feels like it should be "opened" and "closed" to me.

Reply:
I don't think so. Open and closed both describe current state, "opened" is what happened in the past
Example:
The shop is open.
The shop is closed.
The shop was opened.

Reply:
How do you say it was closed in the past then?

Reply:
Yes `open` and `closed` is correct. `closed` is used for both present and past tense in English. Greatest language on Earth! "The shop was closed" and "The shop is closed" and "The shop closed early" are all valid.

Reply:
For open, `opened` is past tense. "The shop opened at 8am" vs "The shop is open" or "The shop will open at 8am"

Reply:
ok great then, thanks everyone :D